### PR TITLE
fix: issue #1786 fixed

### PR DESCRIPTION
**Pull request by sahell0x**
## Description
fix : #1786 

how it is working now :
[Screencast from 2024-10-23 13-10-35.webm](https://github.com/user-attachments/assets/351e42ba-7d65-4e0d-8965-ca78b37cb22d)

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: packages/@sparrow-workspaces/src/hoc/WithModal.svelte [hunk 1]
```diff
@@ -26,9 +26,10 @@
       file?
     </small>
   </div>
-  <div class="d-flex justify-content-between">
-    <div style="margin-right: 15px;">
+  <div class="row">
+    <div class="col-12 mb-2 col-xl-5 col-xxl-3">
```
Comment (itsmdasifraza) [inline packages/@sparrow-workspaces/src/hoc/WithModal.svelte:30]: layout is changing on smaller screens. layout needs to be persistent. all the 3 buttons in a row. may be we can increase modal width.

**Pull request closed without merging**